src/Symfony/Component/ObjectMapper/Transform/MapCollection.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ObjectMapper\Transform;
+
+use Symfony\Component\ObjectMapper\Exception\MappingException;
+use Symfony\Component\ObjectMapper\ObjectMapper;
+use Symfony\Component\ObjectMapper\ObjectMapperInterface;
+use Symfony\Component\ObjectMapper\TransformCallableInterface;
+
+/**
+ * @template T of object
+ *
+ * @implements TransformCallableInterface<object, T>
+ */
+class MapCollection implements TransformCallableInterface
+{
+    public function __construct(private ?ObjectMapperInterface $objectMapper = new ObjectMapper())
```

Comment:
```suggestion
    public function __construct(private ObjectMapperInterface $objectMapper = new ObjectMapper())
```
this can be removed then. Sorry, I forgot in my suggestion